assembl/static2/js/app/utils/translations.js

```diff
@@ -1530,8 +1539,17 @@ const Translations = {
         postSuccess: "Thanks for your participation. Your comment has been sent!",
         postToBeValidated:
           "Your contribution has been saved. To insure the quality of the debate, it will be visible to all participants once it has been reviewed by the animators.",
-        fillSubject: "Please, write a title",
+        postsOrder: {
+          cleanFilter: "Clean filters",
+          chronologicalTop: "Chronological threads",
+          popularityFlat: "Most popular messages first",
+          reverseChronologicalFlat: "Newest messages first",
+          reverseChronologicalLast: "Recently started threads",
```

Comment:
Maybe replace thread for order in general?

Reply:
i follow the specification here